frontend/apps/pwa/src/components/formFields/CheckboxFieldEditor.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
```

Comment:
Does this field not need to be registered?

Reply:
I guess it should still be registered, but without validation. I haven't got to it yet, but I assume we're not using react hook forms for the submission? If we were to do that in the future it'd need to be registered.